stdlib/public/core/StringBridge.swift

```diff
@@ -483,6 +487,15 @@ public func _getDescription<T>(_ x: T) -> AnyObject {
   return String(reflecting: x)._bridgeToObjectiveCImpl()
 }
 
+@_silgen_name("swift_stdlib_boxUTF8CString")
+public func _boxUTF8CString(_ s: UnsafePointer<UInt8>, _ len: Int)
```

Comment:
It'd be better if this weren't `public`. `internal` + `@_silgen_name` ought to be enough, but I'm not sure we made `@_silgen_name` imply "possibly used elsewhere", so you may have to add `@usableFromInline` as well.

Reply:
aha, thank you!

Reply:
@usableFromInline did the trick!